inductiva/fluids/fluid_block/fluid_block.py

```diff
@@ -11,17 +11,20 @@
 from inductiva.fluids.fluid_types import FluidType
 from inductiva.simulators import SPlisHSPlasH
 from inductiva.simulators import DualSPHysics
-from inductiva.utils.templates import replace_params_in_template
+from inductiva.utils import templates
```

Comment:
here we either do just 

```
import inductiva
```
and then expand the whole thing below or, if we assumie that inductiva.utils is going to be something "standard" across we can do semething such as

```
import inductiva.utils as U
```

Please check with @hpenedones but I think moving into a world that we only have high level imports, will make it more obvious when we need to refactor the code (as we already did with scenarios).

Also, please take the chance to fix some class imports if possible.

Reply:
I feel that doing that here, will be a big a pain for testing all of the things. Moreover, I think we should first clear out the importing structure we want to provide, and only afterwards move into these changes.